Linux/Car_Client/gpio.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GPIO_H
+#define GPIO_H
+
+//TODO Check which are used for other things and remove these.
+#define GPIO_PINS 0, 1, 4, 7, 8, 9, 10, 11, 14, 15, 17, 18, 21, 22, 23, 24, 25
+#define STR_LENGTH 64
```

Comment:
These defines shouldn't be in the header since other files will fill up their #defines with random stuff by including it. Rather put it in the .cpp file